config/settings/base.py

```diff
@@ -148,6 +153,25 @@
     },
 }
 
+SENTRY_DEBUG_URL_ENABLED = False
```

Comment:
Can we use SENTRY_DEBUG_URL_ENABLED (line156, line172) in just one place?

Reply:
No, because we want to set the setting value regardless of whether SENTRY_DSN is set